package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.24.0",
+    "millify": "^4.0.0",
```

Comment:
what's this?

Reply:
Axios is Axios;
Millify is a package to facilitate the representation of big numbers. Instead of creating a custom function it already shorter a number. For example 1.4T, 2.25B, 3.54M etc.